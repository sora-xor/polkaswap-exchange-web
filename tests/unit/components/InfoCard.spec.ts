// import { shallowMount, createLocalVue } from '@vue/test-utils'
// import Vuex from 'vuex'

// import InfoCard from '@/components/InfoCard.vue'
// import { SoramitsuElementsImport, TranslationMock } from '../../utils'

// const localVue = createLocalVue()
// localVue.use(Vuex)
// SoramitsuElementsImport(localVue)

// describe('InfoCard.vue', () => {
//   beforeEach(() => {
//     TranslationMock(InfoCard)
//   })

//   it('should renders correctly', () => {
//     const wrapper = shallowMount(InfoCard, {
//       localVue
//     })
//     expect(wrapper.element).toMatchSnapshot()
//   })
// })

describe('InfoCard test', () => {
  test('Temporary test', () => {
    expect(true).toEqual(true);
  });
});
