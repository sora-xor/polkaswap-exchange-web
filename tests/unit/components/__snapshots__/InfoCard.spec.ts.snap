// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoCard.vue should renders correctly 1`] = `
<div
  class="card"
>
  <!---->
   
</div>
`;
